pkg/yurtctl/cmd/revert/edgenode.go

```diff
@@ -123,6 +109,12 @@ func (r *RevertEdgeNodeOptions) Complete(flags *pflag.FlagSet) (err error) {
 	}
 	r.KubeadmConfPath = kubeadmConfPath
 
+	podManifestPath, err := enutil.GetPodManifestPath(r.KubeadmConfPath)
+	if err != nil {
+		return err
+	}
+	r.PodMainfestPath = podManifestPath
```

Comment:
As `podManifestPath` use in func `RemoveYurthub()`, how about remove c.PodManifestPath?

Reply:
updated